app/furniture/marketplace/marketplace.rb

```diff
@@ -30,53 +34,26 @@ def stripe_api_key?
       false
     end
 
-    # @todo Probably want to rename this for-reals but lazy
-    def stripe_account
-      settings["stripe_account"]
-    end
-    alias_method :vendor_stripe_account, :stripe_account
```

Comment:
If we do this, we'll need to migrate from the old setting to this new one somehow, since right now the stripe accounts are stored in the `stripe_account` key of the `settings` hash.

Instead, we may want to do `setting :stripe_account` and keep the alias.

Reply:
I thought we weren't yet at the point where we cared about migrating data for Marketplaces, since it is not used by anyone?

In any case, I reverted this renaming.